### ISSUE TITLE
oxygenite tanks do not upgrade quality with "canned heat" mechanic talent

### ISSUE THREAD

**Issue by doornydew** [Regalis11/Barotrauma#7207]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
oxygenite tanks do not increase in quality with the canned heat talent, yet. high skill does work on it. 

**Steps To Reproduce**
 attempt to make an oxygenite tank with the talent "canned heat" 

**Version**
stable,  windows 10

**Additional information**
-

**Comment by Regalis11** [comment]
Fixed in https://github.com/Regalis11/Barotrauma-development/commit/3f2cef176fc0a9d6729d734b1c9f07540f69505d

**Comment by Rokvach** [comment]
Tested, working correctly. Closing.

**Comment by Udrakan** [comment]
Should 100 mechanical + canned heat make masterwork oxygenite tanks? Because you can't go past Excellent as far as I can test.

**Comment by reformedgamer** [comment]
I am also unable to craft masterwork oxygenite tanks with 100 Mechanical and the Canned Heat talent. In fact, when I reduced my Mechanical skill to a total of 60 with the Canned Heat talent still enabled, I was able to craft Excellent quality Oxygen tanks but only Good quality Oxygenite tanks. It seems the talent is back to being nonfunctional for Oxygenite tanks.
![Barotrauma_JjRqnhwrv4](https://user-images.githubusercontent.com/82622446/162546011-94102993-497a-4a9f-8ee9-e7b8fa7cc247.png)
![Barotrauma_NYyk1hZTAb](https://user-images.githubusercontent.com/82622446/162546028-cf597bdd-71d5-4cf8-a52b-45468f891885.png)
![Barotrauma_EZkrS7FeME](https://user-images.githubusercontent.com/82622446/162546035-5a4a9d42-a06f-4536-928d-8b6e61b0124a.png)
